tests/nested/hot-plug/task.yaml

```diff
@@ -0,0 +1,29 @@
+summary: create ubuntu classic image, install snapd and test hot plug feature
+
+systems: [ubuntu-16.04-64, ubuntu-18.04-64]
+
+prepare: |
+
+    "$TESTSLIB"/prepare-restore.sh --prepare-suite
+    "$TESTSLIB"/prepare-restore.sh --prepare-suite-each
+
+    #shellcheck source=tests/lib/nested.sh
+    . "$TESTSLIB/nested.sh"
+    create_nested_classic_vm
+    copy_remote "${GOHOME}"/snapd_*.deb
+    execute_remote "sudo dpkg -i snapd_*.deb"
```

Comment:
I noticed in my tests that this leave the dependencies on the box partly broken due to ubuntu-core-launcher; I think it would be better to `apt install ./snapd_*.deb` which takes care of that.